app/src/androidTest/java/mobile/programming/musicApp/ExampleInstrumentedTest.kt

```diff
@@ -1,4 +1,4 @@
-package mobile.programming.musicapp
+package mobile.programming.musicApp
```

Comment:
**Technical suggestion #1:** While creating packages, Kotlin naming conventions states that you should try to avoid multi-name package names, such as musicApp. However, if you do use a multi-name package name such as this one, you should use camelcase, which I have changed throughout the project. This information is found in the following link: https://kotlinlang.org/docs/reference/coding-conventions.html#documentation-comments

Reply:
Good noting, I did not think about correct naming conventions when renaming the project for Firebase